src/parsing.jl

```diff
@@ -112,20 +112,24 @@ Base.parse(::Type{T}, pqv::PQValue) where {T} = parse(T, string_view(pqv))
 # allow parsing as a Symbol anything which works as a String
 Base.parse(::Type{Symbol}, pqv::PQValue) = Symbol(string_view(pqv))
 
-function Base.start(pqv::PQValue)
-    sv = string_view(pqv)
-    return (sv, start(sv))
-end
+if VERSION >= v"0.7.0-DEV.5125"
+    Base.iterate(pqv::PQValue, i=1) = iterate(string_view(pqv), i)
```

Comment:
There is no guarantee that the first state of string iteration is `1`, so `iterate(pqv::PQValue)` should pull it from `iterate(string_view(pqv))`. Same is true of the other iteration methods.